lambda-packages/dns-validated-domain-identity-handler/src/verifier.ts

```diff
@@ -155,6 +185,16 @@ export class Verifier {
     };
   }
 
+  private async getCurrentTXTRecord(): Promise<Route53.ResourceRecordSet | undefined> {
+    const res = await this.route53.listResourceRecordSets({
+      HostedZoneId: this.hostedZoneId,
```

Comment:
If you don't want to read entire list, Please supply `StartRecordType` parameter with `TXT` value at least. Someone can have bunch of record sets.

```suggestion
      HostedZoneId: this.hostedZoneId,
      StartRecordType: "TXT",
```